paper.md

```diff
@@ -99,6 +99,45 @@ We also implement some features to improve robustness and flexibility of the ens
 
 * The TOML-file interface defined in the \texttt{UQParameters} module allows non-intrusive use of \texttt{EnsembleKalmanProcesses.jl} through \text{TOML} files, which are widely used for configuration files and easily read in any programming language. Given the computer model to calibrate and prior distributions on the parameters, \texttt{EnsembleKalmanProcesses.jl} reads these distributions from a file and, after an iteration of the ensemble Kalman algorithm, writes each member of the updated ensemble to a parameter file. Each of these parameter files can be then read individually to initiate the ensemble of the computer model for the next iteration.
 
+# Pedagogical example
+
+In this example, the complex computer code simulates a sinusoid curve $$f(A,v) = A\sin(\phi + t) + v, \ \forall t \in [0,2\pi],$$ with a random phase shift $\phi$. We assume that we cannot access $f$, but we can access some informative quantities through the map $$G(A,v) = [\max f(A,v) - \min f(A,v), \mathrm{mean} f(A,v)].$$
```

Comment:
```suggestion
In this example, the computer code simulates a sine wave $$f(A,v) = A\sin(t + \varphi) + v, \ \forall t \in [0,2\pi],$$ with a random phase shift $\varphi$. We assume that we cannot evaluate $f$ directly, but we can access some informative quantities through the map $$G(A,v) = [\max f(A,v) - \min f(A,v), \mathrm{mean} f(A,v)].$$
```
This is more standard notation in natural sciences & engineering.

Reply:
done